server/run_basic_tests.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 cd ./lomas_server
+export PYTHONPATH=$(pwd)/..:$PYTHONPATH
```

Comment:
because all imports are now relative to the package (ie. from lomas_server.utils.constants import ...). so lomas_server must be in the pythonpath if the library is not installed locally (one could do pip install -e alternatively).

Reply:
ok makes sense, thank you